src/test/scala/com/mozilla/telemetry/streaming/TestErrorAggregator.scala

```diff
@@ -278,25 +326,26 @@ class TestErrorAggregator extends FlatSpec with Matchers with BeforeAndAfterAll
     subsession_count should be (10)
   }
 
-  "The aggregator" should "discard non-Firefox pings" in {
+  it should "discard non-Firefox and non-Fennec pings" in {
     import spark.implicits._
-    val fxCrashMessage = TestUtils.generateCrashMessages(k)
-    val fxMainMessage = TestUtils.generateMainMessages(k)
-    val otherCrashMessage = TestUtils.generateCrashMessages(k, Some(Map("appName" -> "Icefox")))
-    val otherMainMessage = TestUtils.generateMainMessages(k, Some(Map("appName" -> "Icefox")))
+    val fxCrashMessages = TestUtils.generateCrashMessages(k)
+    val fxMainMessages = TestUtils.generateMainMessages(k)
+    val otherCrashMessages = TestUtils.generateCrashMessages(k, Some(Map("appName" -> "Icefox")))
+    val otherMainMessages = TestUtils.generateMainMessages(k, Some(Map("appName" -> "Icefox")))
+    val fennecCoreMessages = TestUtils.generateFennecCoreMessages(k)
 
     val messages =
-      (fxCrashMessage ++ fxMainMessage ++ otherCrashMessage ++ otherMainMessage).map(_.toByteArray).seq
+      (fxCrashMessages ++ fxMainMessages ++ otherCrashMessages ++ otherMainMessages ++ fennecCoreMessages).map(_.toByteArray).seq
 
     val df = ErrorAggregator.aggregate(spark.sqlContext.createDataset(messages).toDF, raiseOnError = false,
       ErrorAggregator.defaultDimensionsSchema, ErrorAggregator.defaultMetricsSchema,
       ErrorAggregator.defaultCountHistogramErrorsSchema,
       ErrorAggregator.defaultThresholdHistograms)
 
-    df.where("application <> 'Firefox'").count() should be (0)
+    df.where("application not in ('Firefox','Fennec')").count() should be(0)
   }
 
-  "the aggregator" should "correctly read from kafka" taggedAs(Kafka.DockerComposeTag, DockerErrorAggregatorTag) in {
+  it should "correctly read from kafka" taggedAs(Kafka.DockerComposeTag, DockerErrorAggregatorTag) in {
```

Comment:
Yeah, I hesitated a little over this because this test slowly starts to be hard-to-understand (so I added separate test for Fennec).
On the other hand it's probably worth having one test that has all types of messages - I'll include them and later we can think of some refactoring.